en/lessons/basics/collections.md

```diff
@@ -3,120 +3,224 @@ version: 1.3.1
 title: Collections
 ---
 
-Lists, tuples, keyword lists, and maps.
+Tuples, Lists, Assoc Lists, Sets, Maps/Hashmaps, Seqs, and Vectors.
 
 {% include toc.html %}
 
+## Tuples
+
+Tuples are the first kind of collection you are introduced to in haskell. This is because they are simple, primitive, and have a terse, flexible, built in syntax. Tuples are sometimes referred to as anonymous records, but instead of referencing fields by name you reference them by position in the structure. Tuples can contain an arbitrary number of items, but we will focus on the 2-tuple since haskell has good built in support for them. However, if you are interested this is what an 8-tuple looks like
```

Comment:
1. Can we wrap lines at 80 symbols? It would allow to pinpoint comments and propose suggestions more precisely.
2. Tuples cannot contain an arbitrary number of items, only up to 15-tuple is [required](https://www.haskell.org/onlinereport/basic.html#basic-tuples) by Haskell Report. GHC supports up to 62-tuple.
3. "haskell" -> "Haskell" here and below.